app/assets/javascripts/discourse/app/templates/login.hbs

```diff
@@ -1,176 +1,187 @@
-{{#if
-  (and
-    this.siteSettings.full_page_login
-    (or this.showLogin (not this.siteSettings.login_required))
-  )
-}}
-  {{hide-application-header-buttons "search" "login" "signup" "menu"}}
-  {{hide-application-sidebar}}
-  {{body-class "login-page"}}
-  <div class="login-fullpage">
-    <FlashMessage @flash={{this.flash}} @type={{this.flashType}} />
+{{hide-application-sidebar}}
+{{body-class "login-page"}}
 
-    <div class={{concat-class "login-body" this.bodyClasses}}>
-      <PluginOutlet @name="login-before-modal-body" @connectorTagName="div" />
+{{#if this.isRedirecting}}
+  {{loading-spinner}}
+{{else}}
+  {{#if
```

Comment:
Likely unrelated to this PR, but it would be nice to have a comment here describing when we show the first case of the `if/else` here, and when we show the second. 

I find `and/or` clause here super hard to understand.